docs/Using-Fleet/REST-API.md

```diff
@@ -5134,23 +5140,32 @@ Returns the query specified by ID.
         "platform": "",
         "disabled": false
       }
-    ]
+    ],
+    "stats": {
+      "system_time_p50": 1.32,
+      "system_time_p95": 4.02,
+      "user_time_p50": 3.55,
+      "user_time_p95": 3.00,
+      "total_executions": 3920
+    }
   }
 }
 ```
 
 ### List queries
 
-Returns a list of all queries in the Fleet instance.
+Returns a list of global queries or team queries.
```

Comment:
Just something to be aware of: There's a "slight breaking" API change here (not sure if we can call it breaking though):
Now `GET /api/v1/fleet/queries` will also return "scheduled queries" (whereas currently it returns only the non-scheduled ones).

Reply:
thanks! I think we can live with that.